README.md

```diff
@@ -1,8 +1,7 @@
-# [KARAMBOLAGE](https://markusmcpegamer.github.io/Karambolage "GitHub Webpage")
-### A game by [MarkusMCPEGamer](https://github.com/MarkusMCPEGamer "GitHub") created with the Godot Game Engine
+# [KARAMBOLAGE](https://mister-muffin.github.io/Karambolage/ "GitHub Webpage")
+### A game by [Mister-Muffin](https://github.com/Mister-Muffin "GitHub Profile"), created with the Godot Game Engine
```

Comment:
Wenn schon rewriten, ist es dann so wichtig, in der GitHub README dein Github Profil zu verlinken?

```suggestion
### A game created with the Godot Game Engine
```